### PR TITLE
sriov: update notes for mellanox dpdk

### PR DESCRIPTION
**Pull request by zenghui-shi**

### CODE REVIEW

**Comment by jboxman** [comment]
@zshi-redhat for what other version is this applicable?

**Comment by zshi-redhat** [comment]
> @zshi-redhat for what other version is this applicable?

@jboxman it applies to 4.5, 4.4 and 4.3

**Comment by jboxman** [comment]
@zshi-redhat, for 4.4 and 4.3, is this a doc fix or a change in functionality? If it's just a doc fix, I can merge it. If the functionality actually changed, I'm required to ask support and other stakeholders if it's okay to update the docs.

Does that make sense? Thanks!

**Comment by zshi-redhat** [comment]
> @zshi-redhat, for 4.4 and 4.3, is this a doc fix or a change in functionality? If it's just a doc fix, I can merge it. If the functionality actually changed, I'm required to ask support and other stakeholders if it's okay to update the docs.

Hi Jason, good point! 
It is a doc fix, not adding new functions.
> 
> Does that make sense? Thanks!

Yes, makes sense.

File: modules/nw-sriov-configuring-device.adoc [hunk 1]
```diff
@@ -72,13 +72,14 @@ If you specify both `pfNames` and `rootDevices` at the same time, ensure that th
 +
 [NOTE]
 ====
-For a Mellanox card to work in dpdk mode, use the netdevice driver type.
+For a Mellanox card to work in dpdk mode on Bare metal nodes, use the netdevice driver type and set `isRdma` to `true`.
```
Comment (jboxman) [inline modules/nw-sriov-configuring-device.adoc:75]: `netdevice`

s/Bare metal/bare metal/

Comment (jboxman) [inline modules/nw-sriov-configuring-device.adoc:75]: s/dpdk/DPDK/

File: modules/nw-sriov-configuring-device.adoc [hunk 2]
```diff
@@ -72,13 +72,14 @@ If you specify both `pfNames` and `rootDevices` at the same time, ensure that th
...
+For a Mellanox card to work in dpdk mode with Container-Native Virtualization (CNV), use the vfio-pci driver type and set `isRdma` to `false`.
```
Comment (jboxman) [inline modules/nw-sriov-configuring-device.adoc:76]: s/dpdk/DPDK/

`vfio-pci`

**Review by jboxman — CHANGES_REQUESTED** [review]
@zshi-redhat, added a few comments.

**Comment by jboxman** [comment]
Refs https://github.com/openshift/openshift-docs/pull/22456 because we need a product variable for CNV; Eventually this may need to change to _OpenShift Virtualization_.

File: modules/nw-sriov-configuring-device.adoc (hunk 2 above)
Comment (zshi-redhat) [reply modules/nw-sriov-configuring-device.adoc]: Done

File: modules/nw-sriov-configuring-device.adoc (hunk 1 above)
Comment (zshi-redhat) [reply modules/nw-sriov-configuring-device.adoc]: Done

Comment (zshi-redhat) [reply modules/nw-sriov-configuring-device.adoc]: Done

**Comment by zshi-redhat** [comment]
> @zshi-redhat, looks like this has a merge conflict.

@jboxman thanks for reminding! conflict resolved.

**Review by jboxman — APPROVED** [review]

**Pull request merged**